packages/framework/src/hooks/useMockResponse.ts

```diff
@@ -0,0 +1,13 @@
+const mockResponseTable: Record<string, boolean> = {};
```

Comment:
Yeah let's make this a map from string to object, with the useMockResponse as the first attribute.